views/partials/articles/new.hbs

```diff
@@ -0,0 +1,10 @@
+<h1>Add Article</h1>
+{{!-- articles.item --}}
+<!-- creates a form a user can use to add an item to the list -->
+<form action="/articles/new" method="POST">
+  Title: <input type="text" name="title" value=""><p>
```

Comment:
`<p>` tags require a closing tag. What are you trying to do here create space. You can use <br /> or css